packages/block-editor/src/components/justify-toolbar/index.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
This comments about `External dependencies` should be removed.

Reply:
Removed in 3dab628968